### PR TITLE
fix some grammar rules; re-structure

### PR DESCRIPTION
**Pull request by egorguscha**

### CODE REVIEW

File: docs/api/effector/babel-plugin.md [hunk 1]
```diff
@@ -88,150 +88,151 @@ getUsers.use(
...
 
 :::note since
-effector 20.2.0
+effector 21.6.0
 :::
 
-Option for effector/babel-plugin for making custom unit factories with clean configuration.
+- Type: `string[]`
+
+Accepts an array of module names which exports treat as custom factories therefore each function call provides unique prefix for [sids](./babel-plugin.md#sid) of units inside them. Used to SSR([Server Side Rendering](https://developers.google.com/web/updates/2019/02/rendering-on-the-web)) and it's not required for client-only application.
```
Comment (igorkamyshev) [inline docs/api/effector/babel-plugin.md:106]: I suggest to mention tests here. Or replace mention of SSR by general mentions of Fork API.

File: docs/api/effector/babel-plugin.md [hunk 2]
```diff
@@ -88,150 +88,151 @@ getUsers.use(
...
 }
 ```
 
-```js title="src/state.js"
-import {createInputField} from '@lib/createInputField'
-
-const foo = createInputField('-')
-/*
+```js title="./src/statuses.js"
+import {createEffectStatus} from './createEffectStatus'
```
Comment (igorkamyshev) [inline docs/api/effector/babel-plugin.md:142]: I thought, some bundlers can fail transformation of path here 🤔

The previous version with absolute paths was more stable.

File: docs/api/effector/babel-plugin.md [hunk 3]
```diff
@@ -88,150 +88,151 @@ getUsers.use(
...
+:::note since
+effector 21.5.0
+:::
 
-> `importName: string | string[]`
+- Type: `boolean`
+- Default: `false`
 
-Specify import to process by plugin. By default has value `["effector", "effector/compat"]`
+Replace imports from `effector-react` to `effector-react/ssr`. Useful for building both server-side and client-side builds from the same codebase.
```
Comment (igorkamyshev) [inline docs/api/effector/babel-plugin.md:160]: As I know, `effector-react/ssr` should be replaced with `effector-react/scope` after release 22.

**Review by igorkamyshev — COMMENTED** [review]
Sorry for the unsolicited advices. I have noticed several places where documentation can be improved.

File: docs/api/effector/babel-plugin.md (hunk 2 above)
Comment (egorguscha) [reply docs/api/effector/babel-plugin.md:142]: Do you have an example?

**Pull request merged**